cypress/integration/manualTests/basic-enduser.ts

```diff
@@ -38,58 +111,77 @@ function testWorkflow(url: string, version1: string, version2: string, trsUrl: s
     cy.url().should('contain', '?tab=tools');
   });
 
-  // This tends to be flaky so we are leaving it out for now
-  //
-  // it('DAG tab works', () => {
-  //   goToTab('DAG');
-  //   cy.url().should('contain', '?tab=dag');
-  //   cy.get('[data-cy=dag-holder]');
-  // });
+  it('DAG tab works', () => {
+    cy.contains('.mat-tab-label', 'DAG').click();
+    cy.url().should('contain', '?tab=dag');
+    cy.get('[data-cy=dag-holder]');
```

Comment:
What is this testing? Will it verify there actually is a DAG, or only that there is a container for the DAG?

Just asking because of the removed  code above, where it says "This tends to be flaky". I don't think DAG is flaky (but I could be wrong), I think there are versions it can visualize and versions it can't -- I've never seen  it be flaky for a particular version. 

TLDR; there should be a know version that we can always verify has a DAG (and maybe this is already doing this).

Reply:
Currently, this just checks that the container is there (and added a check for atleast 1 child element).

I believe the flakiness came from the use of [goToTab](https://github.com/dockstore/dockstore-ui2/blob/ba23f699615f627c0897e437d6e4f74f2563669d/cypress/support/commands.ts#L17), as it checks for visibility. Depending on the tabs shown, `DAG` may or may not be visible by default, so skipping the visibility check should make it more reliable.